### PR TITLE
Updates propType expectation to handle a single node or multiples 

### PR DESCRIPTION
**Pull request by mejackreed**
(getting an error without this)

### CODE REVIEW

File: src/components/CollapsibleSection.js [hunk 1]
```diff
@@ -62,7 +62,10 @@ export class CollapsibleSection extends Component {
 }
 
 CollapsibleSection.propTypes = {
-  children: PropTypes.element.isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
```
Comment (cbeer) [inline src/components/CollapsibleSection.js]: I forget; is this maybe equivalent to `PropTypes.element`?

**Comment by jkeck** [comment]
I'm curious what error you're seeing w/ this?  I'm not seeing any errors, but perhaps I'm not looking at the right kind of thing to trigger that.

**Comment by mejackreed** [comment]
@jkeck I'm seeing this on master on initial page load:

Screen Shot 2019-04-10 at 5 45 38 PM

**Comment by jkeck** [comment]
:+1: Ah, I see, it's not an error, it's a warning (which is why I wasn't seeing it in production)

File: src/components/CollapsibleSection.js (hunk 1 above)
Comment (mejackreed) [reply src/components/CollapsibleSection.js]: Yeah I guess so.. pushed up that change.

**Pull request merged**